TODO/effective-okhttp.md

```diff
@@ -161,25 +153,25 @@ WebSettings settings = webView.getSettings();
 settings.setUserAgentString(userAgentHeaderValue);
 ```
 
+### 5\. 指定合理的超时
 
-### 5\. Specify reasonable timeouts
-
-Before version 2.5.0, OkHttp requests defaulted to never timing out. Starting with version 2.5.0, a request times out if establishing a connection, reading the next byte from a connection, or writing the next byte to a connection takes more than 10 seconds to complete. Doing nothing more than updating to version 2.5.0 revealed bugs in our own code, simply because we began exercising certain error paths for the first time. To override these default values, invoke `setConnectTimeout`, `setReadTimeout` or `setWriteTimeout` respectively.
```

Comment:
Doing nothing more than updating to version 2.5.0 revealed bugs in our own code, simply because we began exercising certain error paths for the first time. 
这一句实在是蒙了，感觉可以删掉，没多大意义。